### DISCUSSION TITLE
Column with action buttons in v2

### DISCUSSION THREAD

**Discussion by christian-forgacs** [rappasoft/laravel-livewire-tables#599] [Q&A] [answered]
How I can add buttons with actions in v2? In v1 we can use custom row tables.

**Comment by rappasoft** [comment, marked as answer]
I don't know if i'm adding custom rows in this version yet. There would be too many nuances to implement yourself and there would just be countless issues made because people don't know what to do.

You can still use `->format()` and the new `->view()` in v2 for custom cells though.
